pkg/reporting/cookbooks.go

```diff
@@ -243,9 +245,10 @@ func (cbs *CookbooksStatus) nodesUsingCookbookVersion(cookbook string, version s
 	// TODO add pagination
 	pres, err := cbs.Searcher.PartialExec("node", fmt.Sprintf("cookbooks_%s_version:%s", cookbook, version), query)
 	if err != nil {
-		return nil, errors.Wrap(err, "unable to get cookbook usage information")
+		err = errors.Wrap(err, "unable to get cookbook usage information")
 	}
 
+	// If the error is unrelated to returning any results we want to parse them and return them.
```

Comment:
In order to be able to test the UsageLookupError I needed to return records from this method. @afiune I would love to get your feedback on whether you think this is a good change or not